lib/apipie/errors.rb

```diff
@@ -24,6 +24,20 @@ def initialize(param)
     end
   end
 
+  class ParamMultipleMissing < ParamError
```

Comment:
Would be nice to inherit from the same error as `ParamMissing`, but we do not have the `defined param` here.